skelebot/components/jupyter.py

```diff
@@ -25,11 +25,13 @@ class Jupyter(Component):
 
     port = None
     folder = None
+    mappings = None
```

Comment:
Since we're adding this field we need to add it to the schema validation above as well.